compiler/test/compilable/testcolor.sh

```diff
@@ -15,17 +15,17 @@ compare()
     fi
 }
 
-normalize() { tr -d "\n\r" ; }
+normalize() { tr -d "\n\r" | sed -E 's/void foo\(\) \{\} void main\(\) \{ goo\(\); \}//; s/\s+\^$//'; }
```

Comment:
hmm FreeBSD complains about a backslash (which particular one I'm not sure) here: 
```
==============================
  Test compilable/testcolor.sh failed. The logged output:
  sed: 1: "s/void foo\(\) \{\} voi ...": RE error: trailing backslash (\)
```
not sure why.

Reply:
fixed!